src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.html

```diff
@@ -41,6 +41,14 @@
                 <mat-chip-list>
                   <mat-chip>B</mat-chip> - Baseline submission
                 </mat-chip-list>
+                <mat-chip-list class="pull-right" *ngIf="isChallengeHost">
```

Comment:
Hey @drepram, I think you also need to get the boolean value for `isChallengeHost` in the typescript.